tests/test-cross-context-dispatch/ccd-tests/src/test/resources/plans/context-ee10-forward-dump.json

```diff
@@ -0,0 +1,29 @@
+{
+  "class": "org.eclipse.jetty.tests.ccd.common.DispatchPlan",
+  "requestStep": {
+    "class": "org.eclipse.jetty.tests.ccd.common.steps.HttpRequestStep",
+    "method": "GET",
+    "requestPath": "/ccd-ee10/redispatch/ee10"
+  },
+  "steps": [
+    {
+      "class": "org.eclipse.jetty.tests.ccd.common.steps.ContextRedispatchStep",
```

Comment:
As well as testing a straight `Forward` and `Include` (and then return) between `ee8`, `ee9`, `ee10`, I'd like us to also test the slightly more complex scenarios of :
`Forward -> Forward -> return `, `Forward -> Include -> return `, `Include -> Forward -> return`, `Include -> Include -> return` across `ee8`, `ee9`, `ee10` and soon `ee11`.

Reply:
Sure. Can do.
There's many options we can wire up with this.

Reply:
We also need a test that does:

eeX creates a session, forwards to eeY, which creates a session, returns back to eeX, then eeX forwards again to eeY. Need to check that eeY finds the same session  it previously created.  We also need to check (somehow) that all sessions are correctly completed during the return from dispatch chain.